src/Endpoints/Delegates/HandlesDocuments.php

```diff
@@ -19,23 +29,35 @@ public function addDocuments(array $documents, string $primaryKey = null)
         return $this->http->post(self::PATH.'/'.$this->uid.'/documents', $documents, ['primaryKey' => $primaryKey]);
     }
 
-    public function updateDocuments(array $documents, string $primary_key = null)
+    public function updateDocuments(array $documents, string $primaryKey = null)
     {
-        return $this->http->put(self::PATH.'/'.$this->uid.'/documents', $documents, ['primaryKey' => $primary_key]);
+        return $this->http->put(self::PATH.'/'.$this->uid.'/documents', $documents, ['primaryKey' => $primaryKey]);
     }
 
     public function deleteAllDocuments(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid.'/documents');
     }
 
-    public function deleteDocument(string $document_id): array
+    public function deleteDocument($documentId): array
     {
-        return $this->http->delete(self::PATH.'/'.$this->uid.'/documents/'.$document_id);
+        $this->assertValidDocumentId($documentId);
+
+        return $this->http->delete(\sprintf('%s/%s/documents/%s', self::PATH, $this->uid, $documentId));
     }
 
     public function deleteDocuments(array $documents): array
     {
         return $this->http->post(self::PATH.'/'.$this->uid.'/documents/delete-batch', $documents);
     }
+
+    private function assertValidDocumentId($documentId): void
```

Comment:
This function seems to be really useful. But can you do it in another PR please?

Reply:
Will be handled in https://github.com/meilisearch/meilisearch-php/issues/59